components/Grid/grid.css

```diff
@@ -0,0 +1,25 @@
+:root {
+    --grid-item-bg-color: rgb(212, 243, 212);
+}
+.grid-two-items {
+    border: 1px solid var(--primary-color);
+    display: grid;
+    gap: 10px;
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
Try to use rem in Gaps, helps in reponsiveness

Reply:
As told in the lecture we can use the "px" unit for values up to 10px.